frontend/aml_dashboard/src/index.js

```diff
@@ -540,10 +552,13 @@ create_fiware.$click.subscribe( async () => {
   })
   .then(response => response.json())
   .then(async json => {
-    console.log(json);
-    console.log('CREATED');
 
-    fiware_node_status.$value.set('<h1>Created !</h1> ');
+    if (json.Error) {
```

Comment:
change:
`  fiware_node_status.$value.set(' <h1>Creating...</h1> <br> <img src="https://i.gifer.com/Cad.gif">');` 
to:
`  fiware_node_status.$value.set(' <h2>Creating...</h2> <br> <img src="https://i.gifer.com/Cad.gif">');`